### DISCUSSION TITLE
Possible typo in compute_path() docs - edge_diffraction

### DISCUSSION THREAD

**Discussion by jmoraispk** [NVlabs/sionna#200] [General]
Hello,

In the ([docs of compute_paths()](https://nvlabs.github.io/sionna/api/rt.html#compute-paths)), the argument edge_diffraction (bool) says "If set to **False**, only diffraction on wedges, i.e., edges that connect two primitives, is considered. Defaults to False."

I imagine it was supposed to be "If set to **True**".

Also, which option makes the simulation more precise/realistic?

**Comment by faycalaa** [comment]
Hi,

The API documentation is correct. The flag ``edge_diffraction`` allows for diffraction not just by the wedges, but also by the edges.

Here's a code snippet to demonstrate its use:

```
import sionna
from sionna.rt import load_scene, PlanarArray, Transmitter, Receiver

scene = load_scene(sionna.rt.scene.simple_wedge)
scene.tx_array = PlanarArray(1, 1, 0.5, 0.5, "iso", "V")
scene.rx_array = PlanarArray(1, 1, 0.5, 0.5, "iso", "V")
scene.add(Transmitter("tx", (10.,5.,3.0)))
scene.add(Receiver("rx", (10.,5.,-3.0)))

# Compute diffracted paths only, without diffraction on the edges (default in Sionna)
paths_edges_diff_off =  scene.compute_paths(los=False, reflection=False, diffraction=True, edge_diffraction=False) # No diffraction by edges

# Compute diffracted paths only with diffraction on the edges
paths_edges_diff_on =  scene.compute_paths(los=False, reflection=False, diffraction=True, edge_diffraction=True) # Diffraction by edges is enabled
```

When setting ``edge_diffraction`` to `False`, diffraction occurs exclusively due to the wedge:
![edge_off](https://github.com/NVlabs/sionna/assets/43564757/b489b859-412e-4556-bcd3-78c0016b0d3a)

Setting ``edge_diffraction`` to `True` introduces paths resulting from diffraction on the edges:
![edge_on](https://github.com/NVlabs/sionna/assets/43564757/971f19b8-33bb-41a6-a6d3-04b8feec2911)

Deciding to enable edge diffraction should be based on the characteristics of the specific scene.

**Reply by jmoraispk** [reply]
Thank you for the help once again @faycalaa. 
(Although accurate and technically correct, it's not very intuitive to be to compute diffractions that look like reflections, i.e. incident and diffracted rays with angles < 90º; actually even <180º looks weird... Maybe introducing a "minimum diffraction angle" or something like that would be useful.)
